cnxdb/contrib/pytest.py

```diff
@@ -1,58 +1,83 @@
 # -*- coding: utf-8 -*-
 from __future__ import absolute_import
 import os
-import psycopg2
+
 import pytest
+from sqlalchemy import create_engine
```

Comment:
Hmm, we're jus using this for in-process connection pooling, not ORM, right?

Reply:
Just what is known as 'sqlalchemy core'.